src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -94,6 +97,9 @@ export default class ResultingBusinessName extends Mixins(AmalgamationMixin, Nam
   @Getter(useStore) getNameRequestNumber!: string
   @Getter(useStore) getShowErrors!: boolean
 
+  // Props
+  @Prop({ default: false }) readonly isRegularAmalgamation!: boolean
```

Comment:
Props are useful for "local scope" differences (ie, not in global store), or for components that you want to keep as dumb as possible (ie, you don't want them to depend on anything global like the store or axios).

In this case, this component already has store getters, so just add one more (isAmalgamationFilingRegular). I think this would be cleaner than having to pass the prop in the parent component and extracting it here.

Reply:
Looks good. Resolving.